_posts/2020-04-29-movingpandas-movement-data-analysis.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: single
+title: "MovingPandas: Data Structures and Methods for Movement Data Analysis"
+excerpt: "MovingPandas is an easy to use toolkit for exploring movement data that has recently passed the PyOpenSci review."
+author: "Anita Graser"
+permalink: /blog/movingpandas-movement-data-analysis
+header:
+    overlay_image: images/pandas.jpg
+    overlay_filter: 0.6
+    caption: "Photo credit: [**Ann Batdorf, Smithsonian's National Zoo**](https://www.flickr.com/photos/nationalzoo/5371290900/in/photostream/)"
+categories:
+  - blog-post
+  - pandas
+  - spatial
+---
+
+Movement data is everywhere: from tracking apps that record human or vehicle movement to ecologists monitoring wildlife behaviour. Movement data analysis is challenging since movement data combines time series and spatial data analyses questions. Existing spatial analysis libraries, such as GeoPandas, are great at handling spatial data but they don't support moving objects. 
+
+MovingPandas aims to fill the gap of missing tools for exploring movement data. It provides data structures and methods for dealing with data of moving objects. MovingPandas has been accepted by pyOpenSci as part of its ecosystem in March 2020. 
+
+## A Toolkit for Movement Data Analysis
+
+The MovingPandas repository contains multiple tutorials in the form of Jupyter notebooks that illustrate diverse analysis capabilities using different datasets, including: tracking data of ships, migration of birds, and tracks from a horse's GPS collar. 
+
+MovingPandas Trajectory objects are created from GeoPandas GeoDataFrames. A minimal example would be:
+
+```python
+import pandas as pd
+import geopandas as gpd
+import movingpandas as mpd
+from shapely.geometry import Point
+from datetime import datetime
+from pyproj import CRS
+
+df = pd.DataFrame([
+  {'geometry':Point(0,0), 't':datetime(2018,1,1,12,0,0)},
+  {'geometry':Point(6,0), 't':datetime(2018,1,1,12,6,0)},
+  {'geometry':Point(6,6), 't':datetime(2018,1,1,12,10,0)},
+  {'geometry':Point(9,9), 't':datetime(2018,1,1,12,15,0)}
+]).set_index('t')
+gdf = gpd.GeoDataFrame(df, crs=CRS(31256))
+traj = mpd.Trajectory(gdf, 1)
+```
+
+MovingPandas provides static plots using Matplotlib and interactive plots using hvplot:
+
+```python
+traj.plot()
+```
+
+<img src="https://raw.githubusercontent.com/pyOpenSci/pyopensci.github.io/master/images/movingpandas/mpd_fig1.PNG">
```

Comment:
this image link will need to be adjusted ! 

let's do the following. 
1. make the image extension .png (lower case)
2. image url's should look like this: <img src="{{ site.url }}/images/movingpandas/mpd_fig1.PNG" alt="single sentence text description of image here">
 they will then render properly! 
3. this is optional but adding alt tags would be good as it allows people with screen readers to "see" your images via words" a simple text description is plenty.